README.md

```diff
@@ -129,16 +129,16 @@ There are also some text replacements available which will be replaced by detail
 
 
 ### Static Monitors
-In addition to reading Monitors from Docker labels, AutoKuma can create Monitors from files, this can be usefull if you have want AutoKuma to manage monitors which aren't directly related to a container. 
+In addition to read Monitors from Docker labels, AutoKuma can create Monitors from files. This can be usefull if you have want AutoKuma to manage monitors which aren't directly related to a container. 
```

Comment:
I think reading is actually correct here.

Reply:
"to read" is correct.

Reply:
ChatGPT seems to agree with me:

The correct sentence is:

“In addition to reading Monitors from Docker labels, AutoKuma can create Monitors from files.”

Here’s why: In this context, “reading” is a gerund, which is a verb that functions as a noun in the sentence. The phrase “In addition to” is a preposition that is typically followed by a noun or a gerund (verb-ing). Therefore, “reading” is the correct form to use after “In addition to”. The verb “read” in the second sentence is in the base form, which is not grammatically correct in this context. So, the first sentence is the correct one.